web/client/translations/data.fi-FI.json

```diff
@@ -1195,6 +1195,7 @@
       "showPreview": "Näytä esikatselu",
       "advancedSettings": "Lisäasetukset",
       "templateMetadataAvailable": "Metatiedot saatavissa Dublin Core -muodossa: abstrakti, borderingBox, kontribuuttorit, luojat, kuvaus, formaatti, tunniste, viitteet, oikeudet, lähde, aihe, aika, otsikko, tyyppi, uri",
+      "invalidUrlHttpProtocol": "URL-osoite ei vastaa MapStore HTTP -protokollaa {protocol}",
```

Comment:
We usually translate only the default languages supported by mapstore DE, US, ES, FR and IT so this could be removed